lib/SILOptimizer/SILCombiner/SILCombinerApplyVisitors.cpp

```diff
@@ -600,6 +600,25 @@ SILCombiner::optimizeConcatenationOfStringLiterals(ApplyInst *AI) {
   return tryToConcatenateStrings(AI, Builder);
 }
 
+/// Determine the pattern for global_addr..
+/// %3 = global_addr @$P : $*SomeP
+/// %4 = init_existential_addr %3 : $*SomeP, $SomeC
+/// %5 = alloc_ref $SomeC
+/// store %5 to %4 : $*SomeC
+static SILValue findInitExistentialFromGlobalAddr(GlobalAddrInst *GAI) {
+  /// Check for a single InitExistential.
+  SILInstruction *SingleIE = NULL;
```

Comment:
Two things:

1. This should be nullptr. We do not use NULL in the code base.
2. If you are just looking for InitExistentialAddrInst, then why not just make it this type?